drivers/sensor/sensor_shell.c

```diff
@@ -16,6 +16,12 @@
 	"when no channels are provided. Syntax:\n"                                                 \
 	"<device_name> <channel name 0> .. <channel name N>"
 
+#define SENSOR_ATTR_GET_HELP                                                                       \
+	"Get the sensor's channel attribute. Syntax:\n"                                            \
+	"<device_name> [<channel_name> <attribute_name>].."
```

Comment:
Your implementation  only supports one channel/attribute pair, correct?

```suggestion
	"<device_name> [<channel_name> <attribute_name>]"
```

Reply:
No, you can enter multiple pairs.

Reply:
Discussed offline, I'll add the dynamic commands and make this only 1 channel/attribute pair to keep things simple

Reply:
OK, I was able to toggle channel/attribute using dynamic commands so I'm leaving it as a repeating pair.